tests/sles4sap/redirection_tests/ensa2_kill_sapinstance.pm

```diff
@@ -75,6 +79,7 @@ sub run {
     my @instances_data = @{saphostctrl_list_instances(as_root => 'yes', running => 'yes')};
     my $instance_id = $instances_data[0]->{instance_id};
     my $instance_type = get_instance_type(local_instance_id => $instance_id);
+    my $forced_takeover = ($scenario{forced_takeover} && $instance_type eq 'ASCS') ? '1' : undef;
```

Comment:
is it boolean flag used in two conditions at line 100 and 152? Is there anything better than 1 and undef in perl for bools?